datafusion/core/src/datasource/view.rs

```diff
@@ -114,20 +122,35 @@ impl TableProvider for ViewTable {
                         )
                     })
                     .collect();
-                let plan = LogicalPlanBuilder::from(self.logical_plan.clone())
+                LogicalPlanBuilder::from(self.logical_plan.clone())
                     .project(fields)?
-                    .build()?;
-                state_cloned.create_physical_plan(&plan).await
+                    .build()?
             }
         } else {
-            state_cloned.create_physical_plan(&self.logical_plan).await
+            self.logical_plan().clone()
+        };
+        let mut plan = LogicalPlanBuilder::from(plan);
+        let filter = filters.iter().cloned().reduce(|acc, new| acc.and(new));
```

Comment:
You could use `conjunction` here and above  too if you wanted : https://github.com/apache/arrow-datafusion/blob/master/datafusion/optimizer/src/utils.rs#L123-L147

it just does the same thing, so I don't feel strongly